### ISSUE TITLE
Update order of metadata fields on Work Show Page

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2#764]
### Descriptive summary

The Metadata Team arranged an order for the metadata fields to display in on a work show page: https://docs.google.com/spreadsheets/d/14xLkWt8KI9jCdlvPrNATgORngJEhXKOCoV0LAew1BQ4/edit#gid=1891520168  (that should be on the Final_Order tab but double check)

### Expected behavior

Work show page has metadata fields displayed in the described order.

### Related work

Somewhat related to #143

**Comment by sseymore** [comment]
QA is ready but need PR#899 needs to be merged first.

**Comment by raybrarian** [comment]
#899 has been merged - ready for QA in WC9

**Comment by KevinJonesMeta** [comment]
![image](https://user-images.githubusercontent.com/57114731/72004919-427bff00-3201-11ea-8e7d-ad4ac70993b8.png)

**Comment by KevinJonesMeta** [comment]
Update on list above:

1. Resolved
2. Resolved
4. Extent and Physical Extent resolved. License is displaying twice and has different values in each position (URL in one, label value in other)
5. Location is now displaying but sometimes has geonames URL, and sometimes has label value in field
6. Owner not displaying

I haven't been able to do determine order of fields not visible still. Will discuss with Metadeities how to get this information.

**Comment by raybrarian** [comment]
Moving back to QA until this round of QA is completed

**Comment by raybrarian** [comment]
@KevinJonesMeta - does this one need to go back to the developers to handle 3. and 4. above, or are there other issues that need to be documented before it goes back?

**Comment by KevinJonesMeta** [comment]
This needs to go back to developers to handle 3 and 4 (with the expanded missing fields list). Once that is done I can finish confirming field order.

Related:
Several field labels don't match OD2 Map, listed below.
Should I create a separate ticket for these?

Label displayed that doesn't match OD2 Map (label in map in parentheses)
Ranger District Label (Ranger District)
Water Basin Label (Water Basin)
Publication Place Label (Publication Place)
Place of Production Label (Place of Production)
Date Modified (Modified)
Access Restrictions label (Access Restrictions)

**Comment by CGillen** [comment]
QA:
- [ ] Field labels match MAP.
  - [ ] Ranger District Label (Ranger District)
  - [ ] Water Basin Label (Water Basin)
  - [ ] Publication Place Label (Publication Place)
  - [ ] Place of Production Label (Place of Production)
  - [ ] Date Modified (Modified)
  - [ ] Access Restrictions label (Access Restrictions)
- [ ] Missing fields are now present:
  - [ ] Issued
  - [ ] View Date
  - [ ] Alternative
  - [ ] Tribal Title
  - [ ] Work Type
  - [ ] Hydrologic Unit Code
  - [ ] Location Copy--Shelf Location
  - [ ] Is Part of (This is visible in brief results list; see Herbarium Collection)
  - [ ] Original File Name
- [ ] License only displayed once

**Comment by KevinJonesMeta** [comment]
> QA:
> 
> * [x]  Field labels match MAP.
>   
>   * [x]  Ranger District Label (Ranger District)
>   * [x]  Water Basin Label (Water Basin)
>   * [x]  Publication Place Label (Publication Place)
>   * [x]  Place of Production Label (Place of Production)
>   * [x]  Date Modified (Modified)
>   * [x]  Access Restrictions label (Access Restrictions)
> * [ ]  Missing fields are now present:
>   
>   * [x]  Issued
>   * [ ]  View Date
>   * [x]  Alternative
>   * [x]  Tribal Title
>   * [x]  Work Type
>   * [x]  Hydrologic Unit Code
>   * [x]  Location Copy--Shelf Location
>   * [x]  Is Part of (This is visible in brief results list; see Herbarium Collection)
>   * [x]  Original File Name
> * [x]  License only displayed once

1. View Date is visible on form but not displaying on showpage
2. Orientation is visible on form but not displaying on showpage (I missed this last round)
3. Set not visible on form and not displaying on showpage (I missed this last round)

**Comment by CGillen** [comment]
@KevinJonesMeta 
QA:
- [ ] Missing fields are now present:
  - [ ] View Date
  - [ ] Orientation
Set and OAI Set are two pieces of metadata that are not setup yet. I have some questions for Metadata about what the MAP says regarding those and the OD2 predicate column. I will ask at standups and report on this ticket tomorrow.

**Comment by CGillen** [comment]
Noting here for some amount of quick documentation, and because I said I would.
Set and OAI Set is still having the details hammered out, but it looks like it will use the base Hyrax Collections system. Set is exactly collection and is already on the form (as collection in relationships tab) and the show page (as In Collection on the right column under Collection Information)

OAI Set sounds like it will be a special collection type that a work can only be in one of. It's not likely to be easy to facet or show, but details to come. It would be on the form page already (same as Set above) with some validating we need to build out.
